apps/class-solid/src/lib/experiment_config.ts

```diff
@@ -42,6 +56,28 @@ const jsonSchemaOfExperimentConfigBase = {
       },
       default: [],
     },
+    observations: {
+      type: "array",
+      items: {
+        type: "object",
+        properties: {
+          height: { type: "array", items: { type: "number" } },
```

Comment:
Please add name property

```suggestion
          name: {type: "string"},
          height: { type: "array", items: { type: "number" } },
```